new_members_handler.py

```diff
@@ -18,81 +17,88 @@ def __init__(self, user_id, captcha_message: types.Message, timestamp):
-                                               callback.message.chat.id,
-                                               callback.from_user.id)
-        if pending_user.user_id != callback.from_user.id:
-            await self.bot.answer_callback_query(callback.id,
-                                                 text="We are waiting for " +
-                                                      (await
-                                                       callback
-                                                       .message
-                                                       .chat
-                                                       .get_member(
-                                                           pending_user.user_id
-                                                       )).user.first_name +
-                                                      " to click!",
-                                                 show_alert=False)
+                self.pending_users.remove(pending_user)
+                await pending_user.captcha_message.delete()
+
+    async def handle_captcha_callback(self, callback: types.CallbackQuery):
+        chat_id = callback.message.chat.id
+        message_id = callback.message.message_id
+        from_user_id = callback.from_user.id
+        pending_user_id = self.__get_pending_user(message_id, chat_id).user_id
+        if pending_user_id != from_user_id:
+            await self.bot.answer_callback_query(
+                callback.id,
+                text="We are waiting for {0} to click".format(
+                    (await callback.message.chat.get_member(pending_user_id))
+                    .user.first_name),
+                show_alert=False)
         else:
-            await self.bot.edit_message_text(chat_id=callback.message.chat.id,
-                                             message_id=callback.message.message_id,
-                                             text="Congrats, " + callback.from_user.first_name + ", captcha passed",
-                                             reply_markup=None)
+            await self.bot.edit_message_text(
+                chat_id=chat_id, message_id=message_id,
+                text="Congrats {0}, captcha passed".format(
+                    self.__get_mention(callback.from_user)),
+                parse_mode=ParseMode.HTML, reply_markup=None)
             await self.bot.restrict_chat_member(
-                callback.message.chat.id,
-                callback.from_user.id,
-                types.ChatPermissions(True, True, True, True, True, True, True, True))
-
-    def __get_pending_user(self, message_id, chat_id, user_id):
-        for pendingUser in self.pendingUsers:
-            if message_id == pendingUser.captcha_message.message_id \
-                    and chat_id == pendingUser.captcha_message.chat.id:
-                return pendingUser
+                chat_id, from_user_id,
```

Comment:
looks weird when you have two arguments on one line, and the other argument on another line
change it to this:
```
            await self.bot.restrict_chat_member(
                chat_id,
                from_user_id,
                types.ChatPermissions(True, True, True, True,
                                      True, True, True, True))
```

Reply:
I think it's fine to have that, as long as the indent is right (distinguishable). the third argument is there because it would be look much more ugly to be somewhere else or indent more.